tests/fixtures/security/make-form-login/LoginTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Tests;
+
+use App\Entity\User;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+
+class LoginTest extends WebTestCase
+{
+    public function testLogin(): void
+    {
+        $client = static::createClient();
+        $crawler = $client->request('GET', '/login');
+
+        self::assertResponseStatusCodeSame(200, 'Failed to get \'/login\'.');
+
+        $em = static::getContainer()->get('doctrine')->getManager();
+
+        $user = (new User())
+            ->setEmail('jr@rushlow.dev')
+            ->setPassword('wordpass')
+        ;
+
+        $em->persist($user);
+        $em->flush();
+
+        $form = $crawler->filter('form')->form();
+
+        $form->setValues([
+            '_username' => 'jr@rushlow.dev',
+            '_password' => 'badPass',
```

Comment:
I misread this originally. I thought you were logging in as `jr@rushlow.dev`, password `badass` 😆